tests/functional/test_version_cmd.py

```diff
@@ -3,7 +3,7 @@
 # Taken from the Makefile, to update every time the Makefile version is bumped
 MAJOR = 0
 MINOR = 5
-PATCH = 1
+PATCH = 2
```

Comment:
Can't you get this info from OCR ?

Reply:
It is checked also on the screen in another test. Here we are comparing the response to the get version apdu.